deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,250 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
```

Comment:
Can you update the README.md file at the top of the repo adding `pymatgen` and `matminer` as optional dependencies and installation directions? I think you'll also need to update the `.travis.yml` file so the test environment installs these as well